File: tests/py-files/test-struct1.py
import sys

class Cars:
    def __init__(self, topspeed = 0, acceleration = 0, Tier = 0):
        self.topspeed = topspeed
        self.acceleration = acceleration
        self.Tier = Tier
def main():

    benz = Cars()
    benz.topspeed = 200
    benz.acceleration = 4
    benz.Tier = 'A'
    print("Benz is {} grade and top speed is {}, will reach in {} second".format( benz.Tier,benz.topspeed,benz.acceleration))
    return 0

if __name__ == '__main__':
    main()

File: tests/c-files/test-struct1.c
#include <stdio.h>

struct Cars
{
    int topspeed;
    int acceleration;
    char Tier;
};


int main()
{
    struct Cars benz;
    benz.topspeed = 200;
    benz.acceleration = 4;
    benz.Tier = 'A';
    printf("Benz is %c grade and top speed is %d, will reach in %d second\n", benz.Tier,benz.topspeed,benz.acceleration);

    return 0;
}
